packages/at_secondary_proxy/lib/src/secondary_connection_bridge.dart

```diff
@@ -47,9 +48,9 @@ class SecondaryConnectionBridge {
           var command = utf8.decode(_buffer.getData());
           command = command.trim();
           // If we get to \n and the command doesn't start with "from:" then log something and close this bridge
-          if (! command.startsWith("from:")) {
-            _logger.info('Unexpected command: $command');
-            await _badOpening("Bad form");
+          if (!command.startsWith("from:")) {
+            _logger.info('Looking up secondary for @$command');
+            await _badOpening(proxyUrl);
```

Comment:
Should rename _badOpening since its purpose has now changed. `_writeStringAndClose` is more like what it is doing now

Reply:
Yup was thinking of what to call it and decide to leave it alone as could not come up with a good name.

Will change.

Reply:
changed as suggested